tests/conftest.py

```diff
@@ -79,9 +88,9 @@ def cloned_template(tmp_path_factory):
 
 
 @pytest.fixture()
-def docker() -> LocalCommand:
-    if os.environ.get("DOCKER_TEST") != "1":
-        pytest.skip("Missing DOCKER_TEST=1 env variable")
+def docker(request) -> LocalCommand:
+    if request.config.getoption("--skip-docker-tests"):
+        pytest.skip("Skipping docker tests")
```

Comment:
This change probably means something must change on CI, right?

Reply:
Not really. By default, this won't skip tests, only if you manually add the flag, and the invoke task will only append this flag if explicitly set:
https://github.com/Tecnativa/doodba-copier-template/pull/236/commits/ce62a143eb6717d1e7db15f19756f4fc4b478379#diff-2e746c9eacad6e2c2fdefdc6a665b5ce4607ea384ebf97a729044b79acc778d6R70
https://github.com/Tecnativa/doodba-copier-template/blob/stable/.github/workflows/test.yml#L79

What I can do is remove the `DOCKER_TEST` variable, as it is no longer in use :+1:

Reply:
OK, then remove the variable please.

Reply:
Done :+1: Thanks!